src/test/java/org/candlepin/model/EnvironmentCuratorTest.java

```diff
@@ -54,25 +57,62 @@ public void delete() {
     }
 
```

Comment:
Should also add tests for verifying null/empty inputs (can be done nicely with `@ParameterizedTest` and `@NullAndEmptySource`) and invalid inputs (such as env names and/or types that don't exist).